### ISSUE TITLE
Question on implementation - constant_push?

### ISSUE THREAD

**Issue by kmansoft** [linode/longview#44]
Hello,

I'm making a Go port of Longview client here: https://github.com/kmansoft/longview-go

Have a question about constant_push

What is the meaning of this function? What does it do?

It seems (I added logging) that the client always just sends the most recent snapshot to the server and all historical averaging is handled by the server.

And so data->payload is a one-element array containing only just the most recently collected data item. There is no historical data on the agent side.

Is this correct?

**Comment by kmansoft** [comment]
Another question I have is "dm" and "md" names in disk stats, is that for LVM  / dmcrypt (hope I got these right)?

I'm not familar with that (as a user I mean) at all, always used "direct" partitions myself. Any hints on what the code is doing (for "dm" / "md") would be appreciated.

**Comment by displague** [comment]
https://superuser.com/questions/131519/what-is-this-dm-0-device
https://linux.die.net/man/4/md

**Comment by kmansoft** [comment]
> https://superuser.com/questions/131519/what-is-this-dm-0-device
> https://linux.die.net/man/4/md

Right, thanks, the question was more about what the Linode Longview agent is doing with these, i.e. they're treated somewhat differently from "basic sdX devices", how differently exactly?
